tests/Scripts/RunBasicSearchFunctionalCoreTests.bat

```diff
@@ -6,12 +6,18 @@ cd ..
 REM Configuration
 set config=Release
 set solutionPath="BasicSearchTests.FunctionalTests.sln"
+set exitCode=0
 
 REM Required Tools
 set msbuild="%WINDIR%\Microsoft.NET\Framework\v4.0.30319\msbuild"
 set xunit=".\packages\xunit.runner.console.2.0.0\tools\xunit.console.exe"
```

Comment:
Can we bump to 2.1.0 while you're in here? It supports `-Verbose` which makes debugging easier.